test/Base_Tests/src/Data/Numbers_Spec.enso

```diff
@@ -636,6 +642,9 @@ add_specs suite_builder =
             v1>v2 . should_be_false
             v2>v1 . should_be_true
 
+            v1<=v2 . should_be_true
```

Comment:
@JaroslavTulach For the `<=` and `>=` cases, I get:

```
        Reason: An unexpected panic was thrown: Type error: expected `that` to be Number, but got Complex.
```

Should I change the implementation of `Number.>=`, or is the plan to have some magic in the engine that does an automatic conversion?

Reply:
Here is a fix:
```diff
diff --git engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala
index ad27ed1442..e8db1abd5d 100644
--- engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala
+++ engine/runtime/src/main/scala/org/enso/interpreter/runtime/IrToTruffle.scala
@@ -541,7 +541,7 @@ class IrToTruffle(
                     true
                   )
 
-                val operators = ".!$%&*+-/<>?^~\\"
+                val operators = ".!$%&*+-/<>?^~\\="
                 def isOperator(n: Name): Boolean = {
                   n.name
                     .chars()
```

Reply:
Integrated as 3d5f2c40d712c490b4852724c95a1a8f0363b394 - thanks.